.github/CONTRIBUTING.md

```diff
@@ -20,7 +22,13 @@ We use [StyleLint](https://stylelint.io) to lint our CSS. Our configuration is i
 
 To run StyleLint, use `npm run lint:css`.
 
-### Submitting Issues
+## End-to-End Testing
+
+We use [Jest](https://jestjs.io) for JavaScript Testing.
+
+To run the test, use `npx jest tests/e2e/env_spec.js`.
```

Comment:
I'm really happy if someone improves documentation (I'm really bad in this).

The above command runs only the tests defined in one file `tests/e2e/env_spec.js`. This is perfect for debugging.

The specific test commands are defined in `package.json`, you can run all tests with `npm run test` or specific parts with the other commands, e.g. all unit tests with `npm run test:unit`.

So this should be corrected to "To run all tests, use `npm run test`." Or more detailled as described above.

Reply:
Testing is still new territory for me, so it helps me with learning :slightly_smiling_face: